charts/values-overrides/examples/k3s_example_values.yaml

```diff
@@ -0,0 +1,40 @@
+#example values.yaml of an aws eks based deployment
+tls: false
+graphAppKitPublic: false
+graphAppKitPrivate: false
+#volume pv names go here after initial deployment and provisioning
+volumeName:
+  dataMount: 
+  localMediaMount: 
+  gakPublic: 
+  gakPrivate: 
+
+  ## createServiceAccount: true only true on initial deployment
+k8sDashboard:
+  enabled: true
+  readonly: false
+  createServiceAccount: false
+
+ingress:
+  management:
+    annotations:
+      kubernetes.io/ingress.class: traefik
```

Comment:
this makes sense to do it this way versus hardcoded in other ingresses https://github.com/graphistry/graphistry-helm/blob/c0d809250b556d6ec01cce588d3a6fe5f90e660d/charts/graphistry-helm/templates/ingress/caddy-ingress.yml#L9 so maybe change the other ingresses to reflect this change